otter/cloud_client/rcv3.py

```diff
@@ -242,7 +248,8 @@ def _check_bulk_delete(attempted_pairs, result):
 
         match = _LB_DOESNT_EXIST_PATTERN.match(error)
         if match is not None:
-            del_lb_id = match.group("lb_id")
+            # lb_id returned is in upper case
```

Comment:
If you have to repeat this comment all over the place, I'd just make a function `normalize_lb_id` and then give it a docstring that explains.